README.md

```diff
@@ -39,6 +39,11 @@ This is the mono-repo for Manta/Calamari/Dolphin nodes.
   cargo b --profile production
   ```
 > Tips: The binary will be generated under `target/production/manta`. For less performance critical build, `cargo build --release` is recommended for faster build time.
+3. Run standalone dev chain, useful for testing
+  ```bash
+  cargo run -- --chain=dolphin-localdev --alice --force-authoring --tmp
```

Comment:
iirc we discussed with @zqhxuyuan that force-authoring is not needed

Reply:
Yep good catch, it doesn't do anything

Reply:
Works the same without it